backend/src/north/north-oianalytics/north-oianalytics.ts

```diff
@@ -43,23 +39,26 @@ export default class NorthOIAnalytics extends NorthConnector<NorthOIAnalyticsSet
   }
 
   override async testConnection(): Promise<void> {
-    const connectionSettings = await this.getNetworkSettings(`/api/optimistik/oibus/status`);
-    const requestUrl = `${connectionSettings.host}/api/optimistik/oibus/status`;
-    const fetchOptions: RequestInit = {
+    const host = this.getHost();
+    const requestUrl = `${host}/api/optimistik/oibus/status`;
+    const proxyOptions = this.getProxyOptions();
+    const authOptions = await this.getAuthorizationOptions();
+
+    const fetchOptions: ReqOptions = {
       method: 'GET',
-      headers: connectionSettings.headers,
-      timeout: this.connector.settings.timeout * 1000,
-      agent: connectionSettings.agent
+      auth: authOptions,
+      proxy: proxyOptions,
+      timeout: this.connector.settings.timeout * 1000
```

Comment:
I don't think we need these intermediate variables, we can directly build and pass the options in the HTTPRequest call